CHANGELOG.md

```diff
@@ -1,9 +1,69 @@
 # Changelog
 
-
 ## [Unreleased](https://github.com/confio/poe-contracts/tree/HEAD)
 
-[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.5.4...HEAD)
+[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.6.0-beta1...HEAD)
+
+## [v0.6.0-alpha1](https://github.com/confio/poe-contracts/tree/v0.6.0-beta1) (2022-02-03)
+
+[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.6.0-alpha1...v0.6.0-beta1)
+
+**Breaking changes:**
+
+- `valset`: Optimize VALIDATORS storage [\#42](https://github.com/confio/poe-contracts/issues/42)
+- Optimize validators storage [\#61](https://github.com/confio/poe-contracts/pull/61) ([maurolacy](https://github.com/maurolacy))
+
+**Closed issues:**
+
+- Voting's helper `update_status()` should save changed status [\#62](https://github.com/confio/poe-contracts/issues/62)
+- \[voting-contract\] More status issues [\#55](https://github.com/confio/poe-contracts/issues/55)
+
+**Merged pull requests:**
+
+- Voting Contract: vote and close uses current status [\#60](https://github.com/confio/poe-contracts/pull/60) ([ueco-jb](https://github.com/ueco-jb))
+
+## [v0.6.0-alpha1](https://github.com/confio/poe-contracts/tree/v0.6.0-alpha1) (2022-01-31)
+
+[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.5.5...v0.6.0-alpha1)
+
+**Breaking changes:**
+
+- \[voting-contract\] Record proposal creator [\#31](https://github.com/confio/poe-contracts/issues/31)
+
+**Closed issues:**
+
+- Validator Voting: check `migrate_msg` content to be \>0 [\#46](https://github.com/confio/poe-contracts/issues/46)
+- Add Open Text Proposals [\#15](https://github.com/confio/poe-contracts/issues/15)
+
+**Merged pull requests:**
+
+- Update rust toolchain to v1.58.1 [\#58](https://github.com/confio/poe-contracts/pull/58) ([uint](https://github.com/uint))
+- 0.6.0-alpha1 release [\#56](https://github.com/confio/poe-contracts/pull/56) ([uint](https://github.com/uint))
+- Voting contract: save info about creator of proposal [\#54](https://github.com/confio/poe-contracts/pull/54) ([ueco-jb](https://github.com/ueco-jb))
+- Fix `remove_hook` helper [\#53](https://github.com/confio/poe-contracts/pull/53) ([maurolacy](https://github.com/maurolacy))
+- Validator set query pagination [\#51](https://github.com/confio/poe-contracts/pull/51) ([maurolacy](https://github.com/maurolacy))
```

Comment:
Are we backward-nitpicky?:)
This is breaking.

Reply:
Good point. If we all go and add `breaking` to our breaking issues in that list, I can easily regenerate the CHANGELOG.